app/controllers/api/scratchpad_pages_controller.rb

```diff
@@ -0,0 +1,17 @@
+module API
+  class ScratchpadPagesController < BaseController
+    def update
+      page = current_user.scratchpad_pages.find_by(uuid: params[:id])
+
+      return render_404 unless page
```

Comment:
@iHiD When do we decide to use a generic 404 message versus using a specific message (i.e. "Scratchpad isn't found")

Reply:
We always want a specific message really. The generic is the "lazy" option for when we don't know what the exact failed lookup is (say we get an uncaught `ActiveRecord::RecordNotFound`)

Reply:
Cool. I've added a specific message for this now.